test/dotnet-projectmodel-server.Tests/DthTests.cs

```diff
@@ -422,7 +422,7 @@ public void WaitForLockFileReleased(int occupyFileFor, bool expectSuccess)
             {
                 var lockFilePath = Path.Combine(testProject, LockFile.FileName);
                 var lockFileContent = File.ReadAllText(lockFilePath);
-                var fs = new FileStream(lockFilePath, FileMode.Create, FileAccess.Write, FileShare.None);
+                var fs = new FileStream(lockFilePath, FileMode.Open, FileAccess.Write, FileShare.None);
```

Comment:
if the open mode worth test then both mode should be added not change from one to another.